NuGet.config

```diff
@@ -26,6 +26,7 @@
     </packageSource>
     <packageSource key="dotnet-public">
       <package pattern="*" />
+      <package pattern="Microsoft.Orleans.*" />
```

Comment:
Can we remove the orleans-nightly?

Reply:
If we use it again, we can add it again. Might as well remove things we aren't using.

Reply:
Agreed. Removed.